### PR TITLE
FC-900 | Fix light appearance of loading spinner

### PR DESCRIPTION
**Pull request by zoheiry**
## Purpose
- Light appearance for loading spinner was using a color that didn't exist.
- Also the way that opacity was added to the hex color doesn't work with the shorthand hex code (3 characters only) So i used hexToRgb function instead.

#### Before
screen shot 2018-11-26 at 2 49 18 pm
#### After
screen shot 2018-11-26 at 2 49 34 pm

### Links
- [Jira Ticket](https://fashiontrade.atlassian.net/browse/FC-900)

### Checklist
- [x] Update changelog

### CODE REVIEW

File: src/components/Button/Button.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ const ButtonInner = ({ size, appearance, children, ...otherProps }) => (
 );
 
 ButtonInner.propTypes = {
-  children: PropTypes.string.isRequired,
+  children: PropTypes.node.isRequired,
```
Comment (zoheiry) [inline src/components/Button/Button.js]: Fixing warning.

File: src/components/Input/Input.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
 Empty:
 
 ```jsx
-<Input type="text" placeholder="Empty state" value="" />
+<Input type="text" placeholder="Empty state" value="" onChange={() => null} />
```
Comment (zoheiry) [inline src/components/Input/Input.md]: Also fixing warnings.

File: src/components/LoadingSpinner/LoadingSpinner.js [hunk 3]
```diff
@@ -10,7 +11,7 @@ const colors = {
   success: defaultTheme.success,
   danger: defaultTheme.danger,
   warning: defaultTheme.warning,
-  light: defaultTheme.grey06,
```
Comment (zoheiry) [inline src/components/LoadingSpinner/LoadingSpinner.js]: Color didn't exist.

Comment (kirill-kruchkov) [reply src/components/LoadingSpinner/LoadingSpinner.js]: My bad! Sorry about that

File: src/components/LoadingSpinner/LoadingSpinner.js [hunk 4]
```diff
@@ -27,7 +28,7 @@ const StyledSpinner = styled.span`
 
 const LoadingSpinner = ({ size, speed, appearance, ...otherProps }) => {
   const defaultColor = colors[appearance];
-  const defaultGapColor = `${defaultColor}25`;
+  const defaultGapColor = hexToRgba(defaultColor, 0.25);
```
Comment (kirill-kruchkov) [inline src/components/LoadingSpinner/LoadingSpinner.js]: Just to double check.. is transparency a valid case here? Can't we use an existing colour from our theme?

Comment (zoheiry) [reply src/components/LoadingSpinner/LoadingSpinner.js]: The design is currently that any loading spinner has a gap color that is 25% lighter than the original color, I don't think we have a light version of every appearance to achieve the same result.

**Pull request merged**